index.html

```diff
@@ -225,6 +225,57 @@ <h4>Install the client on your headset</h4>
+                                </li>
+                                <li class="nav-item" role="presentation">
+                                    <button class="nav-link" id="bitrate-tab" data-bs-toggle="tab" data-bs-target="#bitrate" type="button" role="tab" aria-controls="bitrate" aria-selected="true">Bitrate</button>
+                                </li>
+                                <li class="nav-item" role="presentation">
+                                    <button class="nav-link" id="enconders-tab" data-bs-toggle="tab" data-bs-target="#encoders" type="button" role="tab" aria-controls="encoders" aria-selected="false">Encoders</button>
+                                </li>
+                            </ul>
+                            <div class="tab-content" id="myTabContent2">
+                                <div class="tab-pane fade show active" id="foveation" role="tabpanel" aria-labelledby="foveation-tab">
+                                    <p>A stronger foveation makes the image sharper in the center than in the periphery and makes the decoding faster. This is better for fast paced games. </p>
+                                    <p>A weaker foveation gives a uniform sharpness in the whole image.</p>          
+                                    <p>The recommended values are between 20% and 50% for headsets without eye tracking and between 50% and 70% for headsets with eye tracking.</p>
+                                    <img src="screenshots/settings/foveation.jpg" class="d-block w-100">
+                                </div>
+                                <div class="tab-pane fade" id="bitrate" role="tabpanel" aria-labelledby="bitrate-tab">
+                                    <p>Bitrate of the video, in bit/s. Split among decoders based on size and codecs.</p>
+                                    <p>A higher bitrate directly improves image quality by reducing pixelation in fast-paced scenes or in complex visuals, such as landscapes with dense foliage. However, it may also increase latency or overload the network, encoder, or decoder.</p>
+                                    <p>For testing purposes, a bitrate of 50 Mbps is a good starting point for wireless connections. Wired connections, if supported by the hardware, should be able to handle up to 500 Mbps.</p>
+                                    <img src="screenshots/settings/bitrate.jpg" class="d-block w-100">
+                                </div>
+                                <div class="tab-pane fade" id="encoders" role="tabpanel" aria-labelledby="encoders">
+                                    <p>Encoders compresses video data in real-time to reduce bandwidth and latency, enabling smooth transmission to the headset.</p>
+                                    <p>Default value: 3 encoders in order 1/8th, 3/8th and 4/8th of the image.</p>
```

Comment:
We should move the information about defaults after explaining what is configurable